python/Ice/greeter/server_asyncio/main.py

```diff
@@ -13,7 +14,7 @@ async def main():
 
     # Create an Ice communicator. We'll use this communicator to create an object adapter. We enable asyncio
     # support by passing the current event loop to initialize.
-    async with Ice.initialize(eventLoop=loop) as communicator:
+    async with Ice.initialize(sys.argv, eventLoop=loop) as communicator:
```

Comment:
We should rename this server in a follow-up PR. Since everything uses asyncio, we should not name a server "server_asyncio".